src/hooks/v2/NftRewards.ts

```diff
@@ -26,16 +28,22 @@ async function getRewardTierFromIPFS({
 
   const response = await axios.get(url)
   const ipfsRewardTier: IPFSNftRewardTier = response.data
+  const maxSupply = contractNftRewardTier.initialQuantity.eq(
```

Comment:
Gets `maxSupply` and `contributionFloor` attributes directly from contract rather than IPFS now